docs/source/_templates/objective_class.rst

```diff
@@ -0,0 +1,37 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+   {% set class_attributes = ['name', 'greater_is_better', 'perfect_score', 'positive_only', 'problem_types',
```

Comment:
How come we need to set class_attributes here if we're going to manually add them one-by-one below?

Reply:
You're right, it's absolutely unnecessary, let me remove it!